modules/caddyhttp/templates/tplcontext.go

```diff
@@ -160,6 +160,7 @@ func (c TemplateContext) funcHTTPInclude(uri string) (string, error) {
 	}
 	virtReq.Host = c.Req.Host
 	virtReq.Header = c.Req.Header.Clone()
+	virtReq.Header.Add("Accept-Encoding", "identity")
```

Comment:
@mholt Indeed, it has been treat as additional value(can't be overwritten) if `Accept-Encoding` already have other value so that it is suitable to use `Set()`. I'll fix it.
https://pkg.go.dev/net/http#Header.Add